src/components/WorkOrder/TimeLog.js

```diff
@@ -1,17 +1,25 @@
 import React from "react";
+import { Link } from "react-router-dom";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import { faSpinner } from "@fortawesome/free-solid-svg-icons";
+import { faSpinner, faEdit } from "@fortawesome/free-solid-svg-icons";
 import { FIELDS } from "./formConfig";
 
-const TimeLog = ({ data }) => {
+const TimeLog = ({ data, workOrderId }) => {
   return (
     <div>
       {data.length === 0 && <p>No data</p>}
       {data.length > 0 && (
         <ul className="list-group list-group-flush">
           {data.map((timeLog, i) => (
             <li className="list-group-item d-flex row" key={i}>
-              <div className="col-7">
+              <div className="col-1 mt-2">
+                <Link
+                  to={`/work-order/${workOrderId}/edit-time-log/${timeLog.id}`}
+                >
+                  <FontAwesomeIcon icon={faEdit} size="2x" />
```

Comment:
Could we make this be on the right side of the page instead of the left? And also, let's use a button with text + icon. I think that will be more consistent. See the Edit button above on the same page:

![Screen Shot 2019-09-24 at 1 32 52 PM](https://user-images.githubusercontent.com/5697474/65541075-3b3f1880-ded2-11e9-8884-536c75a47a25.png)

Don't mind using the blue color for the background though.

Reply:
👍 I like this a lot better. I forgot about our trusty `<Button />` component which made this an easy change.

![Screen Shot 2019-09-24 at 2 59 08 PM](https://user-images.githubusercontent.com/37249039/65546027-4008ca00-dedc-11e9-8367-34502b4e8eda.png)